ExternalActor

chronaeon opened the pull request.

webthethird commented:
Rather than `using signed transactions` we might say `using signed messages, including transactions and contract calls`, in order to better align with #91

webthethird commented:
In the case of private blockchains and off-chain state channels, wouldn't the external actor be the software client which, when triggered by some encoded logic on the external network, creates the actual signed message on the Ethereum blockchain? In other words, should there be a distinction made between external actors and external protocols? Let's get some feedback from @jo-tud

chronaeon commented:
Good point! I think that distinction is very important!

On Mon, Jun 19, 2017 at 12:04 PM, William E Bodell III <
notifications@github.com> wrote:

> In the case of private blockchains and off-chain state channels, wouldn't
> the external actor be the software client which, when triggered by some
> encoded logic on the external network, creates the actual signed message on
> the Ethereum blockchain? In other words, should there be a distinction made
> between external actors and external protocols? Let's get some feedback
> from @jo-tud <https://github.com/jo-tud>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ConsenSys/EthOn/pull/98#issuecomment-309486624>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AUAJt3gkcd6HZ9XcKZVHmxfscJslLOQhks5sFpwPgaJpZM4N82jk>
> .
>



-- 
Micah D. Dameron

chronaeon commented:
I don't think we even need to wait for feedback from @jo-tud on that because it's right on! Will you edit the definition to reflect that and I'll go over it a final time before resubmitting?

I could edit it myself, but since it's your good idea I think it will come out better if you're saying it.

chronaeon commented:
Alright I added your clarification, thanks Bill!

I removed the latter 2/3rds of the definition, as it was too long and strayed too far outside of a definition IMO!
